.travis.yml

```diff
@@ -18,74 +16,57 @@ env:
 before_install:
-    # Add the modern toolchain repositories (llvm repo depends on ubunut-toolchain-r + gcc)
-    - sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
-    - sudo add-apt-repository -y 'deb http://llvm.org/apt/precise/ llvm-toolchain-precise-3.8 main'
-    - wget -O - http://llvm.org/apt/llvm-snapshot.gpg.key | sudo apt-key add -
-
     # Get the latest CMake
     - curl -O https://cmake.org/files/v3.2/${CMAKE_VERSION}.tar.gz
 
-    # MongoDB Enterprise Edition, latest stable version
-    - sudo apt-key adv --keyserver ${KEY_SERVER} --recv 7F0CEB10
-    - sudo apt-key adv --keyserver ${KEY_SERVER} --recv D68FA50FEA312927
-    - echo "deb ${MONGO_REPO} ${REPO_TYPE}" | sudo tee ${SOURCES_LOC}
-
-    # CMake
-    - sudo add-apt-repository --yes ppa:kalakris/cmake
-
-    # Update all the repositories
-    - sudo apt-get update -qq
-
-    # Mongo C Driver
-    - curl -O -L https://github.com/mongodb/mongo-c-driver/releases/download/${CDRIVER_VERSION}/mongo-c-driver-${CDRIVER_VERSION}.tar.gz
-
-    # Mongo C++ Driver
-    - git clone -b 'master' --single-branch https://github.com/mongodb/mongo-cxx-driver.git
-
 install:
     # GCC
-    - if [ "$CXX" = "g++" ]; then sudo apt-get install -qq g++-6; export CXX="g++-6"; export CC="gcc-6"; export NO_ERROR_ON_MAYBE_UNINITIALIZED="-Wno-error=maybe-uninitialized"; fi
+    - if [ "$CXX" = "g++" ]; then export CXX="g++-6"; export CC="gcc-6"; export NO_ERROR_ON_MAYBE_UNINITIALIZED="-Wno-error=maybe-uninitialized"; fi
 
     # Clang
-    - if [ "$CXX" == "clang++" ]; then sudo apt-get install --allow-unauthenticated -qq clang-3.8; export CXX="clang++-3.8"; export CC="clang-3.8"; fi
+    - if [ "$CXX" == "clang++" ]; then export CXX="clang++-3.8"; export CC="clang-3.8"; fi
 
     # Don't Wno-error-maybe-uninitialized if we're in a Debug build
     - if [ "$CONFIG" == "Debug"]; then export NO_ERROR_ON_MAYBE_UNINITIALIZED=""; fi
 
     # CMake
     - tar -zxf ${CMAKE_VERSION}.tar.gz -C build
 
-    # Install MongoDB Enterprise
-    - sudo apt-get install mongodb-enterprise-server
-
-    # Install CMake (Versions lower than 2.8.8 do not support object libraries)
-    - sudo apt-get install cmake
-
     # Prepare pkg-config path
-    - export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:/home/travis/install/lib/pkgconfig/
-
-    # Install Mongo C Driver
-    - tar -zxf mongo-c-driver-${CDRIVER_VERSION}.tar.gz
-    - pushd mongo-c-driver-${CDRIVER_VERSION}; ./configure --prefix=/home/travis/install; make; sudo make install; popd;
+    - export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:$HOME/deps/lib/pkgconfig/
 
-    # Install Mongo C++ Driver
-    - pushd mongo-cxx-driver/build; ../../build/${CMAKE_BINARY} -DCMAKE_CXX_FLAGS="-Wno-ignored-attributes" -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/home/travis/install/ ..; make; sudo make install; popd; 
+    # Install dependencies
+    - git clone -b 'master' --single-branch https://github.com/mongodb/mongo-cxx-driver.git
```

Comment:
Yes, because it contains the FindLibBSON.cmake file that needs to be found by CMAKE_MODULE_PATH.